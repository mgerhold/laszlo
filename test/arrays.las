let numbers = [1, 2, 3];
println(numbers); // [1, 2, 3]
for i in 0..3 {
    numbers[i] = numbers[i] * 2;
}
println(numbers); // [2, 4, 6]

let words = ["some", "words"];
println(words); // [some, words]
words[1] = "stuff";
println(words); // [some, stuff]

let to_be_mutated = ["Hello, ", "Greetings, ", "Good day, "];
for word in to_be_mutated {
    word = word + "Santa Claus!";
}
println(to_be_mutated);
