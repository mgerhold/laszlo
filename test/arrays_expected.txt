[1, 2, 3]
[2, 4, 6]
[some, words]
[some, stuff]
[Hello, Santa Claus!, Greetings, Santa Claus!, Good day, Santa Claus!]
